test_conformance/mem_host_flags/checker_image_mem_host_write_only.hpp

```diff
@@ -256,7 +256,7 @@ cl_int cImage_check_mem_host_write_only<T>::verify_RW_Image_Mapping()
     test_error(err, "clReleaseEvent error");
 
     // Map image for writing
-    T* dataPtr = (T*)clEnqueueMapImage(
```

Comment:
Think the formatting changes in this file are out of scope for this PR

Reply:
indeed, `git clang-format` is culprit. Do you want me to revert it somehow ? The same thing happened with `command_buffer_profiling` branch